qucs/qucs-transcalc/transline.h

```diff
@@ -25,6 +25,8 @@
 
 class QucsTranscalc;
 
+const double c0 = 299792458;// Speed of light (m/s)
```

Comment:
you're going too fast, leave some time for reviewing to others also...
Nothing bad here at a first sight, but things like needing to define `c0` *should* smell bad; it's in fact already defined [here](https://github.com/Qucs/qucs/blob/9d924caa3879fbca978987a9fe964eadbc5f0fc4/qucs/qucs-transcalc/units.h#L35) (but at least it has the same value :grin:) and used in many other places.